test/helper_test.py

```diff
@@ -98,6 +98,15 @@ def http_response_mock(body="", headers=None, status=200):
     return HTTPResponse(body, HTTPHeaderDict(headers), status=status)
 
 
+def provisioning_response_mock(body='{"foo":"bar"}', headers={"foo": "bar"}, status=200):
```

Comment:
`headers={"foo": "bar"}` is dangerous, since dict is mutable, you might get unexpected behaviour.